Vinyl/HTTPURLResponse.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  HTTPURLResponse.swift
+//  Vinyl
+//
+//  Created by Rui Peres on 17/02/2016.
+//  Copyright © 2016 Velhotes. All rights reserved.
+//
+
+import Foundation
+
+// Idea taken from VENMO/DVR
+class HTTPURLResponse: NSHTTPURLResponse {
+    
+    private var _statusCode: Int?
+    override var statusCode: Int {
```

Comment:
Instead of doing this I think we can manage it by using this initializer from `NSHTTPURLResponse`:

```swift
init?(URL url: NSURL, statusCode: Int, HTTPVersion: String?, headerFields: [String : String]?)
```